cfg/cfg.go

```diff
@@ -103,10 +103,13 @@ type Route struct {
 	Topic         string // also used by Google PubSub
 	Codec         string // also used by Google PubSub
 	PartitionBy   string
-	TLSEnabled    bool   `toml:"tls_enabled"`
-	TLSSkipVerify bool   `toml:"tls_skip_verify"`
-	TLSClientCert string `toml:"tls_client_cert"`
-	TLSClientKey  string `toml:"tls_client_key"`
+	TLSEnabled    bool
```

Comment:
the convention for the toml representation (config file) is to use lower case and underscored.